setup.py

```diff
@@ -123,9 +123,9 @@
     classifiers=[
         "License :: OSI Approved :: Apache Software License",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 3.6",
```

Comment:
Great! We've finally said goodbye to python 3.6!

Reply:
We've actually disabled CI for 3.6 for a while, forgot to remove it here :P